src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -501,6 +544,26 @@ public void updateSubscribers()
     }
 
     /**
+     * Sends packages to update the fields.
+     *
+     * @param oldSubscribers    the existing subscribers.
+     * @param hasNewSubscribers the new subscribers.
+     */
+    private void sendFieldPackets(Set<EntityPlayerMP> oldSubscribers, boolean hasNewSubscribers)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "oldSubscribers". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1172)